edx_proctoring/utils.py

```diff
@@ -393,3 +394,7 @@ def categorize_inaccessible_exams_by_date(onboarding_exams, details):
                 non_date_inaccessible_exams.append(onboarding_exam)
 
     return non_date_inaccessible_exams, future_exams, past_due_exams
+
+
+# Signal that is emitted when an attempt status is updated. Added to utils to avoid cyclic import in signals.py file
```

Comment:
this seems weird, what's the cycle?

Reply:
signals.py imports edx_proctoring.api, so I can't import edx_proctoring.signals into api.py

Reply:
feels like a cleanup ticket for proctoring/signals and this signal is in order. Though maybe after we remove 

signals uses api.ProctoredExamStudentAttemptSerializer which does not need to be in api
and it uses update_attempt_status which will be thornier to untangle

Reply:
Yeah, I had tried to get rid of the cyclic import but it seemed a little out of scope for this ticket. I can definitely file a follow up ticket for clean up though. 

I found it odd that we were importing ProctoredExamStudentAttemptSerializer from `api.py` as opposed to `serializers.py`, but wasn't sure if there was a reason for that.